web/src/reducers/budget.js

```diff
@@ -150,9 +150,12 @@ const buildBudget = incomingBigState => {
       activity.statePersonnel.push(
         ...keyPersonnel
           .filter(kp => kp.hasCosts)
-          .map(({ costs }) => ({
+          .map(({ costs, fte }) => ({
             years: Object.keys(costs).reduce(
-              (o, ffy) => ({ ...o, [ffy]: { amt: costs[ffy], perc: 1 } }),
+              (o, ffy) => ({
+                ...o,
+                [ffy]: { amt: costs[ffy], perc: fte[ffy] }
+              }),
```

Comment:
I expected the math part of this to be more complicated, but it looks like also changing this to a by-year FTE property, it's actually a pretty straightforward change. Nice!